<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="outputTB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>410</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
	border: 3px solid #ffe57d;
	border-radius: 10px; 
	color: #ac6e23;
	background-color: #ffebff;
	padding: 10px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}
</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ff4aa8;
font: 550 14pt &quot;MS Shell Dlg 2&quot;;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Output (Cipher Text)</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: #ffd4ff;
}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ff4aa8;
font: 500 18pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>My Own Cipher</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="textInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>410</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 1px solid #ffd4ff;
	color: #ac6e23;
	background-color: #ffebff;
	border-radius: 5px; 
	padding: 10px;
}

QPlainTextEdit:hover {
	border: 3px solid #ffe57d;
}
</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="inputBut">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>410</width>
      <height>120</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 500 12pt &quot;MS Shell Dlg 2&quot;;
	border: #ac6e23;
	border-left 2px solid transparent;
	border-radius: 15px; 
	background-color: #ffd4ff;
	color: #ff66e3;
}

QPushButton:hover {
	background-color: #ffa5ea;
}

QPushButton:pressed {
	background-color: #ff4aa8;
	color: #ffffff;
}
</string>
    </property>
    <property name="text">
     <string>Insert file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ff4aa8;
font: 550 14pt &quot;MS Shell Dlg 2&quot;;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Plain Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cryptBut">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>550</y>
      <width>410</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 500 20pt &quot;MS Shell Dlg 2&quot;;
	border: none;
	border-left 2px solid transparent;
	border-radius: 15px; 
	background-color: #ffd4ff;
	color: #ff66e3;
}

QPushButton:hover {
	background-color: #ffa5ea;
}

QPushButton:pressed {
	background-color: #ff4aa8;
	color: #fff;
}
</string>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ff4aa8;
font: 550 14pt &quot;MS Shell Dlg 2&quot;;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="keyInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>411</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 1px solid #ffd4ff;
	color: #ac6e23;
	background-color: #ffebff;
	border-radius: 5px; 
	padding: 10px;
}

QPlainTextEdit:hover {
	border: 3px solid #ffe57d;
}
</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="enigmaBut">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>690</y>
      <width>410</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 700 11pt &quot;MS Shell Dlg 2&quot;;
	border: none;
	border-left 2px solid transparent;
	border-radius: 15px; 
	background-color: #ffffff;
	border: 3px solid #ffd4ff;
	color: #ff66e3;
}

QPushButton:hover {
	background-color: #ffa5ea;
	border: none;
}

QPushButton:pressed {
	background-color: #ff4aa8;
	color: #fff;
}

</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ff4aa8;
font: 500 11pt &quot;MS Shell Dlg 2&quot;;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ff4aa8;
font: 500 11pt &quot;MS Shell Dlg 2&quot;;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>1001</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff8e1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: #ebebeb;
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
	padding-left: 10px;
	border: 2px solid #ffdca0;
	background-color: #fff8e1;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>Plain Text</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Text File</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Binary File</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ff4aa8;
font: 550 14pt &quot;MS Shell Dlg 2&quot;;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Input Type</string>
    </property>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>outputTB</zorder>
   <zorder>label_3</zorder>
   <zorder>widget</zorder>
   <zorder>textInput</zorder>
   <zorder>inputBut</zorder>
   <zorder>label_2</zorder>
   <zorder>cryptBut</zorder>
   <zorder>label_4</zorder>
   <zorder>keyInput</zorder>
   <zorder>enigmaBut</zorder>
   <zorder>radioButton</zorder>
   <zorder>radioButton_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
